.travis.yml

```diff
@@ -28,12 +28,7 @@ jobs:
     if: type = push AND branch = master
     # set env to avoid deploy matrix
     env: TEST=helm
-    script:
-      - openssl aes-256-cbc -K $encrypted_d8355cc3d845_key -iv $encrypted_d8355cc3d845_iv -in travis.enc -out travis -d
```

Comment:
I copied ci/deploy.sh from zero-to-jupyterhub, but forgot to replace the values with the ones here for this repo.

Reply:
Maybe a sign we should pass the variable as parameters if we want to keep things in sync.

Reply:
+1 @Carreau